server/sonar-web/src/main/js/apps/quality-profiles/profile-details-view.js

```diff
@@ -84,9 +84,9 @@ export default Marionette.LayoutView.extend({
       format (item) {
         return item.name;
       },
-      searchUrl: '/api/qualityprofiles/projects?key=' + encodeURIComponent(key),
-      selectUrl: '/api/qualityprofiles/add_project',
-      deselectUrl: '/api/qualityprofiles/remove_project',
+      searchUrl: window.baseUrl + '/api/qualityprofiles/projects?key=' + encodeURIComponent(key),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)